src/figuresClasses.ts

```diff
@@ -1,11 +1,94 @@
-export interface Figure {}
+'useStrict';
```

Comment:
The 'useStrict' directive is incorrectly written as 'useStrict' instead of 'use strict'. It should be 'use strict' to enable strict mode in JavaScript.